README.md

```diff
@@ -1 +1,47 @@
 # Conduit Connector Azure Storage
+
+## General
+
+The Azure Storage plugin is one of [Conduit](https://github.com/ConduitIO/conduit) plugins.
+It currently provides only source connector.
+
+## How to build it
+
+Run `make`.
+
+## Source
+
+The Source connector monitors given Azure Blob container for file changes and generates appropriate Record when change is detected.
+It supports two reading modes and switches them automatically:
+- Snapshot
+- CDC
+
+In **Snapshot mode**, connector reads the current state of the container, meaning it does not include changes made during this process.
```

Comment:
If snapshot iterator gets interrupted, it starts over again. Works the same as S3 snapshot iterator, I believe.

When switched to CDC iterator, it will detect all changes made while snapshot iterator was active.